### ISSUE TITLE
PSNR is not finite when training on waymo

### ISSUE THREAD

**Issue by jinyummiao** [dvlab-research/BlockNeRFPytorch#34]
Hi, many thanks for your efforts in these works!
I run the training code on the waymo dataset following your instructions and modified some configuration to save the CPU and GPU memory (chunk size 100, batch size 512). However, a problem arises that the training metrics are not finite and a debugging process stops the training code.

In Pdb, the dictionary `metric` contains four key-value pair, i.e., `psnr`, `depth_variance`, `photo_loss` and `loss`, but their values are all `nan`. And the values `item['rgbs']` and `item['rays]` seem proper.

![image](https://user-images.githubusercontent.com/38695019/189521391-ce84cf9a-0269-4bab-813f-8b089d954bf3.png)
![image](https://user-images.githubusercontent.com/38695019/189521407-3b0232e7-192c-46dd-a344-fef014d38be3.png)
![image](https://user-images.githubusercontent.com/38695019/189521423-a589f7c3-3809-4ab2-9c0d-02ac65c40af4.png)

Looking forward to your reply! Thank you at advance!

**Comment by sjtuytc** [comment]
Thanks for your attention. It seems that this is an issue of the official Mega-NeRF repo. Many community members are facing this issue. https://github.com/cmusatyalab/mega-nerf/issues/35
A member in our community is handling this issue and I'm working on the Block-NeRF algorithm part.

**Comment by sjtuytc** [comment]
If you address this, welcome to PR.

**Comment by jinyummiao** [comment]
After resetting the chunk size to 200, the same problem arose on 17-th training step. Have you met the same problem? and how do you solve it? It seems that you have already successfully trained Block-NeRF model on waymo according to your updated README.

And it has been noticed that the problem is randomly appeared in the training step. The second try shows that the training process stops on 82-th step due to the same reason.

**Comment by sjtuytc** [comment]
The Mega NeRF algorithm still gets stuck here. But we @SEUleaderYang made progress on the Block-NeRF algorithm (on the development branch). 
I think this issue is due to some parameters, I will look into this later if our community friend could not address this issue.

**Comment by jinyummiao** [comment]
I try to just ignore the training step that throw this error and it doesn't seem to affect the whole training process currently LOL. And I'm still looking forward to your progress on such problem.

**Comment by jinyummiao** [comment]
It seems the problem is solved in the updated version of the repository. Thanks for your efforts, it helps us a lot!

**Comment by ghaodong** [comment]
> 

我一直在关注你们对于大场景重建的工作，我最近在用mega-nerf训练自己的数据集，出现了psnr=nan这种情况，请问目前有什么好的解决方法吗？我再mega-nerf社区并未找到答案，想在您这边寻求点帮助

**Comment by sjtuytc** [comment]
Mega NeRF已经不再被我们使用了

**Comment by ghaodong** [comment]
我看您们现在主要集成了block-nerf，你们接下来会主要集成哪些模型到社区呀



| |
***@***.***
|
|
***@***.***
|




---- 回复的原邮件 ----
| 发件人 | Zelin ***@***.***> |
| 日期 | 2023年05月09日 19:50 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [sjtuytc/UnboundedNeRFPytorch] PSNR is not finite when training on waymo (Issue #34) |

Mega NeRF已经不再被我们使用了

**Comment by sjtuytc** [comment]
不会集成了，这不是我们的目的

**Comment by ghaodong** [comment]
想问一下你们这个社区接下来主要做哪个方向呀



| |
***@***.***
|
|
***@***.***
|




---- 回复的原邮件 ----
| 发件人 | Zelin ***@***.***> |
| 日期 | 2023年05月09日 20:23 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [sjtuytc/UnboundedNeRFPytorch] PSNR is not finite when training on waymo (Issue #34) |

不会集成了，这不是我们的目的

**Comment by sjtuytc** [comment]
不太清楚

**Comment by lhc991025** [comment]
> 似乎问题已在存储库的更新版本中解决。感谢您的努力，这对我们有很大帮助！

ray_altitude_range ！
How do you get the value of this parameter when you train your own dataset?
